release/device-snmp-go.yaml

```diff
@@ -1,18 +1,18 @@
 ---
 name: 'device-snmp-go'
-version: '1.3.1'
-releaseName: 'hanoi'
+version: '2.0.0'
+releaseName: 'ireland'
 releaseStream: 'master'
-commitId: '2835f428d5bb359d09284af55213ecc07555b5ca'
+commitId: '828c04b1f0eb937037ce2076f410b90c0d25fe4c'
```

Comment:
I see `189d47ff213bf991ba18b7c83461bde848db16b3` as the latest commit on `master`.